crates/cairo-lang-casm-contract-class/Cargo.toml

```diff
@@ -12,7 +12,7 @@ cairo-lang-utils = { path = "../cairo-lang-utils", default-features = false }
 num-bigint.workspace = true
 serde.workspace = true
 itertools = { workspace = true, default-features = false }
-thiserror-no-std.workspace = true
+thiserror.workspace = true
```

Comment:
I'm not sure it is the good fix.
Did you try this:
```rust
[features]
default = ["std"]
std = [
   ...,
   thiserror-no-std/std, // <------
]

Reply:
I tried it now and it works! However, I have a doubt. Does this give the wrong idea that the crate supports no_std when it actually does not?

Reply:
Some of them does. `cairo-lang-casm-contract-class` is one of them :)

Reply:
So yeah, if the crate support no-std use this-error-no-std, if not keep thiserror

Reply:
oh yes, I missed that my bad, fixing it

Reply:
so it seems `cairo-lang-casm-contract-class` didn't need thiserror so I have removed that dependency. so none of the `no_std` crates depend on thiserror now